File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/TagMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.dto.objects.TagDTO;
import pl.yalgrin.playnite.simplesync.library.domain.Tag;

@Component
public class TagMapper extends AbstractObjectMapper<Tag, TagDTO> {

    @Override
    protected TagDTO createDTO() {
        return new TagDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/TagMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Tag
import pl.yalgrin.playnite.simplesync.library.dto.TagDTO

@Component
class TagMapper : LibraryObjectMapperImpl<Tag, TagDTO>() {
    override fun createDTO(): TagDTO = TagDTO()
}